### PR TITLE
Player properly logged on an EntityBlockFormEvent

### LINKED ISSUE #218: [BUG] FrostedIce created by a player does not log the player name.
**Describe the bug**
Frosted ice formed by a player does not log the player name, but instead that the entity `player` created it. 

**To Reproduce**
1. Wear boots with Frost Walker on them
2. Walk on water.
3. Preform a lookup on the ice.

**Expected behavior**
The username of the player who forms the ice, should be logged to the database.

**Screenshots**
screenshot

**Server (please complete the following information):**
  - Prism Version: 2.1.6-127
  - Server Type: Paper
  - Server Version: 1.16.2

**Additional context**
I believe this is caused by [this line](https://github.com/AddstarMC/Prism-Bukkit/blob/master/src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java#L762). It gets the entity type name and passes that to `ActionFactory.createBlockChange()`. For players, it should cast the entity to an `OfflinePlayer` and pass that to `ActionFactory.createBlockChange()`. I'm working on implementing this, any guidance on alternative fixes is also appreciated.

### PR DESCRIPTION
**Pull request by joeyahines**
+ When the player is the cause of BlockFormEvent, the player is now logged instead of the entity type of `player`
+ Fixes #218

Any feedback is appreciated!

### CODE REVIEW

File: src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java [hunk 1]
```diff
@@ -759,10 +760,17 @@ public void onEntityBlockForm(final EntityBlockFormEvent event) {
...
         final BlockState newState = event.getNewState();
-        final String entity = event.getEntity().getType().name().toLowerCase();
 
-        RecordingQueue.addToQueue(ActionFactory.createBlockChange("entity-form", loc, block.getType(),
-                block.getBlockData(), newState.getType(), newState.getBlockData(), entity));
+        if (event.getEntity() instanceof Player) {
+            final OfflinePlayer player = (OfflinePlayer)event.getEntity();
+            RecordingQueue.addToQueue(ActionFactory.createBlockChange("entity-form", loc, block.getType(),
+                    block.getBlockData(), newState.getType(), newState.getBlockData(), player));
+        }
```
Comment (Narimm) [inline src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java:768]: Codacy found an issue: ['}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6131844)

**Comment by Narimm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6131844&bid=20054049)

File: src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java [hunk 2]
```diff
@@ -759,10 +760,16 @@ public void onEntityBlockForm(final EntityBlockFormEvent event) {
         final Block block = event.getBlock();
         final Location loc = block.getLocation();
         final BlockState newState = event.getNewState();
-        final String entity = event.getEntity().getType().name().toLowerCase();
 
-        RecordingQueue.addToQueue(ActionFactory.createBlockChange("entity-form", loc, block.getType(),
-                block.getBlockData(), newState.getType(), newState.getBlockData(), entity));
+        if (event.getEntity() instanceof Player) {
+            final OfflinePlayer player = (OfflinePlayer)event.getEntity();
```
Comment (viveleroi) [inline src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java]: You can cast to `Player` and avoid an extra import.

**Review by Narimm — APPROVED** [review]

**Comment by Narimm** [comment]
Thanks @joeyahines

**Pull request merged**